kinto/core/authorization.py

```diff
@@ -12,11 +12,9 @@
 logger = logging.getLogger(__name__)
 
 # A permission is called "dynamic" when it's computed at request time.
+# XXX: this is the only one we have, is that necessary?
 DYNAMIC = "dynamic"
 
-# When permission is set to "private", only the current user is allowed.
-PRIVATE = "private"
```

Comment:
We do use PRIVATE :dancer:

Reply:
`
./kinto_chefclub/views/devices.py:
    6 : from kinto.core.authorization import PRIVATE
   51 : @devices.delete(permission=PRIVATE)
   68 :     permission=PRIVATE, schema=PayloadSchema(), validators=(colander_body_validator,)
`